modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -253,106 +262,57 @@ public void startRaftGroupNode(
         try {
             List<Peer> peers = nodes.stream().map(n -> new Peer(n.address())).collect(Collectors.toList());
 
-            String locNodeName = clusterNetSvc.topologyService().localMember().name();
+            LOG.info("Start new raft node for group={} with initial peers={}", grpId, peers);
 
-            if (deltaNodes.stream().anyMatch(n -> locNodeName.equals(n.name()))) {
-                LOG.info("Start new raft node for group={} with initial peers={}", grpId, peers);
+            if (!raftServer.startRaftGroup(grpId, raftGrpEvtsLsnr, lsnr, peers, groupOptions)) {
```

Comment:
something is wrong, you don't even use delta nodes

Reply:
That's right, I check that node is "delta" outside of the method, just like it should have been done in the first place.

Reply:
Yeah, I should probably remove the argument then, thank you!